schema.sql

```
@@ -190,6 +190,61 @@ CREATE TABLE IF NOT EXISTS insider_transactions (
+    positions integer NOT NULL DEFAULT 0,
+    positions_skipped integer NOT NULL DEFAULT 0,
+    total_return real,
+    annualized_return real,
+    sharpe real,
+    hit_rate real,
+    benchmark_total_return real,
+    excess_return real,
+    params_json text,
+    created_at timestamptz DEFAULT now()
+);
+CREATE INDEX IF NOT EXISTS idx_backtest_runs_strategy ON backtest_runs (strategy, created_at);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)